virttest/migration.py

```diff
@@ -456,19 +456,20 @@ def check_result(self, result, params):
             if int(result.exit_status) != 0:
                 raise exceptions.TestFail(result.stderr_text.strip())
 
-    def do_cancel(self):
+    def do_cancel(self, sig=signal.SIGKILL):
         """
         Kill process during migration.
 
+        :param sig: The signal to send
         :raise: test.error when kill fails
         """
         def _get_pid():
-            cmd = "ps aux |grep 'virsh .* migrate' |grep -v grep |awk '{print $2}'"
+            cmd = "ps aux |grep 'virsh .*migrate' |grep -v grep |awk '{print $2}'"
             pid = process.run(cmd, shell=True).stdout_text
             return pid
 
         pid = utils_misc.wait_for(_get_pid, 30)
-        if utils_misc.safe_kill(pid, signal.SIGINT):
+        if utils_misc.safe_kill(pid, sig):
```

Comment:
What is the reason to replace signal.SIGINT with signal.SIGKILL? Does it impact existing tp-libvirt cases?

Reply:
Actually, it's a method which hasn't been called yet.  It will be used in new automation case 2663. 
The previous signal  SIGINT cannot stop migration correctly and checking on vm after cancelling will failed in this case. So update the default value to SIGKILL instead and make it configurable.